package.json

```diff
@@ -67,25 +67,25 @@
   },
   "devDependencies": {
     "coveralls": "~2.11.6",
-    "grunt-concurrent": "~2.0.0",
-    "grunt-contrib-copy": "~0.8.0",
-    "grunt-contrib-csslint": "~0.4.0",
-    "grunt-contrib-cssmin": "~0.12.3",
-    "grunt-contrib-jshint": "~0.11.2",
+    "grunt-concurrent": "~2.2.1",
```

Comment:
as to when we are dropping it that's still open, and we can't leave those out of date packages hanging there. Let's update, whenever we drop that's fine.